conf/config.yaml

```diff
@@ -88,6 +88,7 @@ plugins:                          # plugin list
   - openid-connect
   - proxy-rewrite
   - redirect
+  - response-rewrite
```

Comment:
the plugin name `response` is fine

Reply:
I think  it better to indicate one plugin use the same name
 while name is already defined in response-rewrite.lua
`plugin_name = "response-rewrite"` 

if we change it in `conf/config.yaml` , it wont pass the test case

Reply:
you are right